setup.py

```diff
@@ -43,7 +43,7 @@
     "types-PyYAML",
     "types-simplejson",
     "types-requests",
-    "setuptools==65.6.3;python_version>='3.7'",
+    "setuptools==70.0.0;python_version>='3.7'",
```

Comment:
Hello, using setuptools version 65.6.3 is causing this [vulnerability](https://nvd.nist.gov/vuln/detail/cve-2022-40897). Will it be possible to merge this PR to address that? Thanks!